### ISSUE TITLE
Changing default localization give issue

### ISSUE THREAD

**Issue by tmdan** [thedevdojo/voyager#5091]
## Version information
 - Laravel: v7.9.2
 - Voyager: v1.4
 - PHP: 7.4
 - Database: Postgresql 12

## Description
After changing the language in the multilingual -> default cell in the configurations voyager.php file, the tabs for switching the language stop working correctly when adding and when viewing data in CRUD table.

## Steps To Reproduce
1. Add some data on few different language in table which support and realise multilanguage trait.
2. Check it it work correct.
3. Go to voyager.php config file and change default language in multilingual field.
4. Go back in crud table view and try to change language between supported languages this will not work, although the data was entered correctly before.

## Additional context
You can watch on video the issue.
https://youtu.be/FJv0HDW4pj8.

**Comment by MrCrayon** [comment]
It's not a bug it's just how translations work in Voyager.
Let's say you have a BREAD posts with title translatable, default language RU is saved in `posts.title` while for translation a row in `translations` is created/updated with `table_name` `posts`, `locale` `en` etc.
When you change default to EN you are telling Voyager that what is in `posts.title` is EN and needs to find RU translation in `translations` table.
Since there is no translation RU in `translations` table it will fallback to default EN so even when you switch language you will always see EN (that in reality is still your RU sentence).

You can verify that is still working going on edit and setting the missing translation.

**TLDR**
Do not change default language after you already inserted your translations.
